src/main-root.css

```diff
@@ -1,13 +1,38 @@
 /**
- * 1. Set the font family of the entire document to use 'system-ui'.
- * 2. Set the font size of the entire document using the default user-agent size
+ * Set a more readable tab size.
+ *
+ * ✌︎ Credit https://github.com/sindresorhus/modern-normalize/issues/17
+ */
+
+:root {
+	tab-size: 4;
```

Comment:
not sure why, but the indenting on this seems quite massive compared to the rest of the code.  

Spooky considering it relates to `tab-size`

Reply:
Not seeing anything untoward in my editor.